build.gradle.kts

```diff
@@ -10,7 +10,7 @@ plugins {
 }
 
 group = "uk.nhs.hee.trainee.details"
-version = "0.33.2"
+version = "1.0.0"
```

Comment:
I mean, technically it is a breaking change :trollface:

Reply:
Can't break a pre-release, but yeah it's overdue 😂